ods_ci/tests/Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot

```diff
@@ -357,15 +357,13 @@ Add And Run JupyterLab Code Cell 5 In Active Notebook
     [Arguments]    @{code}    ${n}=1
     # This keyword was copied and amended from JupyterLibrary resources - Notebook.Add And Run JupyterLab Code Cell.
 
-    ${add icon} =    Get JupyterLab Icon XPath Custom    add
-
     ${nb} =    Get WebElement    xpath://div${JLAB XP NB FRAG}\[${n}]
     ${nbid} =    Get Element Attribute    ${nb}    id
 
     ${active-nb-tab} =    Get WebElement    xpath:${JL_TABBAR_SELECTED_XPATH}
     ${tab-id} =    Get Element Attribute    ${active-nb-tab}    id
 
-    Click Element    xpath://div[@aria-labelledby="${tab-id}"]/div[1]//${add icon}
+    Click Element    xpath://div[@aria-labelledby="${tab-id}"]//div[@data-jp-item-name="insert"]
```

Comment:
@jstourac doesn't this overlap with your fix?

Reply:
Hm, yeah, very well spotted, Luca. These two changes in this file should be reverted as we can use the approach before with the update of the global values for the Jupyter based on the version in use.

Reply:
@FedeAlonso do you mind reverting it just to be sure I'm not missing anything that could break on your side?

Reply:
... well, I went on and raised #1905 then.